packages/patternfly-4/react-core/src/components/Pagination/Pagination.d.ts

```diff
@@ -0,0 +1,17 @@
+import { FunctionComponent, HTMLProps, ReactNode, ReactElement } from 'react';
+import { Omit } from '../../typeUtils';
+
+export const PaginationVariant: {
+  top: 'top';
+  bottom: 'bottom';
+  left: 'left';
+  right: 'right';
+};
+
+export interface PaginationProps extends HTMLProps<HTMLDivElement> {
+
```

Comment:
Needs props

Reply:
Um, yeah, that would be good. Nice catch!